packages/search-in-workspace/src/browser/search-in-workspace-preferences.ts

```diff
@@ -58,6 +58,11 @@ export const searchInWorkspacePreferencesSchema: PreferenceSchema = {
             description: nls.localizeByDefault('Controls whether to follow symlinks while searching.'),
             default: true,
             type: 'boolean',
+        },
+        'search.additionalRoots': {
+            description: nls.localizeByDefault('Directories outside the current workspace that should be included in searches.'),
```

Comment:
I do not believe there is a default value for this string in `nls.metada.json`:

```suggestion
            description: nls.localize('theia/search-in-workspace/additionalRoots' 'Directories outside the current workspace that should be included in searches.'),
```